lib/rules/jsx-max-depth.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * @fileoverview Validate JSX maximum depth
+ * @author Chris<wfsr@foxmail.com>
+ */
+'use strict';
+
+let variableUtil = require('../util/variable');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Validate JSX maximum depth',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    schema: [{
+      oneOf: [
+        {
+          type: 'integer',
```

Comment:
I'd prefer to not even allow this as an option, if possible - meaning, to only have the object schema.